src/main/java/com/example/onlinebookstore/dto/book/BookDto.java

```diff
@@ -1,14 +1,17 @@
 package com.example.onlinebookstore.dto.book;
 
 import java.math.BigDecimal;
+import java.util.List;
+import lombok.Data;
 
-public record BookDto(
-        Long id,
-        String title,
-        String author,
-        String isbn,
-        BigDecimal price,
-        String description,
-        String coverImage
-) {
+@Data
```

Comment:
Why did you decide to use lombok instead of record?

Reply:
I do not remember. However, something could not work with my record, probably it was not a problem with it, but I did not change it

Reply:
so let’s  return record))

Reply:
I then lose the method setCategoryIds  in BookMapper

Reply:
Does `@Mapping(source = "categories", target = "categoriesId") `annotation in BookMapper help in this case?

Reply:
while I am not sure that I need it, because everything is working correctly at the moment

Reply:
Ok. You can just try both of options )